### PR TITLE
html structure added

### PR DESCRIPTION
**Pull request by ryliebutnerWSU**
html structure added, but most content is placeholder at the moment.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              <li><a href="index.html"><h2>Rylie Butner</h2></a></li>
+              <li><a href="work.html">Work</a></li>
+              <li><a href="misc.html">Misc.</a></li>
+              <li><a href="contact.html">Contact</a></li>
+            </ul>  
+        </nav>
+    </header>
+    <main>
+        <h1>Contact Form</h1>
+    <form class="cf">
```
Comment (axamei) [inline contact.html:21]: We will go into forms later in the semester. Labels are required so keep that in mind

File: index.html [hunk 2]
```diff
@@ -3,9 +3,41 @@
...
+              <li><a href="work.html">Work</a></li>
+              <li><a href="misc.html">Misc.</a></li>
+              <li><a href="contact.html">Contact</a></li>
+            </ul>  
+        </nav>
+    </header>
+    <main>
+        <div><img src="images/clip-art.jpg" alt="Me as clip art"></div>
+        <div>
+            <h4>Design | Librarianship</h4>
```
Comment (axamei) [inline index.html]: I recommend using a span. We want to use headings and their appropriate levels to communicate the hierarchy of the content on the page. It is also an accessibility issue to skip heading levels. See: https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

**Review by axamei — CHANGES_REQUESTED** [review]
Hi @ryliebutnerWSU it looks like we're missing h1s on some pages that have other heading levels. Please review the comments and make the requested changes.

File: index.html (hunk 2 above)
Comment (ryliebutnerWSU) [reply index.html]: Thank you, this makes a lot of sense! I was making the mistake on choosing to use headings based on the size of text in my design.

File: contact.html (hunk 1 above)
Comment (ryliebutnerWSU) [reply contact.html:21]: Got it, I've added labels.

**Review by axamei — APPROVED** [review]
👍

**Pull request merged**